lib/cmd/fh3/appforms/environments/submissions/create.js

```diff
@@ -0,0 +1,40 @@
+var fs = require('fs');
+var _ = require('underscore');
+
+module.exports = {
+  'desc' : 'Create A New Submission',
+  'examples' : [{ cmd : 'fhc appforms environments submissions create --environment=<Environment Id> --submissiondata=<Path To JSON Submission Data> --projectid=<project to associate the submission with>', desc : 'Create A New Submission'}],
+  'demand' : ['environment', 'submissiondata'],
+  'alias' : {},
+  'describe' : {
+    'environment': "Environment Id",
+    'submissiondata': "Path To JSON Submission Data",
+    'projectid': "Project Guid To Associate The Submission With"
+  },
+  'url' : function(params){
+    return "/api/v2/mbaas/" + params.environment + "/appforms/submissions";
+  },
+  'preCmd' : function(params, cb){
+    var submissionDataPath = params.submissiondata;
+
+    //Reading The File Passed For Form Content
+    fs.readFile(submissionDataPath, function(err, submissionFileContent){
```

Comment:
as below, maybe consider checking for err as well as relying on try/catch handler?